apps/web/src/app/(main)/(mdx)/privacy/page.mdx

```diff
@@ -18,96 +20,113 @@ Sensitive Information. We do not process sensitive information.
 
-## 6\. IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?
+
+## 6. IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?
 
 _In Short: We may transfer, store, and process your information in countries other than your own._
 
 Our servers are located in. If you are accessing our Services from outside, please be aware that your information may be transferred to, stored, and processed by us in our facilities and by those third parties with whom we may share your personal information (see "WHEN AND WITH WHOM DO WE SHARE YOUR PERSONAL INFORMATION?" above), in and other countries.
 
 If you are a resident in the European Economic Area (EEA), United Kingdom (UK), or Switzerland, then these countries may not necessarily have data protection laws or other similar laws as comprehensive as those in your country. However, we will take all necessary measures to protect your personal information in accordance with this privacy notice and applicable law.
 
-## 7\. HOW LONG DO WE KEEP YOUR INFORMATION?
+
+## 7. HOW LONG DO WE KEEP YOUR INFORMATION?
 
 _In Short: We keep your information for as long as necessary to fulfill the purposes outlined in this privacy notice unless otherwise required by law._
 
-We will only keep your personal information for as long as it is necessary for the purposes set out in this privacy notice, unless a longer retention period is required or permitted by law (such as tax, accounting, or other legal requirements).
+We will only keep your personal information for as long as it is necessary for the purposes set out in this privacy notice, unless a longer retention period is required or permitted by law (such as tax, accounting, or other legal requirements). 
 
 When we have no ongoing legitimate business need to process your personal information, we will either delete or anonymize such information, or, if this is not possible (for example, because your personal information has been stored in backup archives), then we will securely store your personal information and isolate it from any further processing until deletion is possible.
 
-## 8\. DO WE COLLECT INFORMATION FROM MINORS?
+
+## 8. DO WE COLLECT INFORMATION FROM MINORS?
 
 _In Short: We do not knowingly collect data from or market to children under 18 years of age._
 
 We do not knowingly solicit data from or market to children under 18 years of age. By using the Services, you represent that you are at least 18 or that you are the parent or guardian of such a minor and consent to such minor dependent’s use of the Services. If we learn that personal information from users less than 18 years of age has been collected, we will deactivate the account and take reasonable measures to promptly delete such data from our records. If you become aware of any data we may have collected from children under age 18, please contact us at [support@keyshade.xyz](mailto:support@keyshade.xyz) .
```

Comment:
Update this `We do not knowingly solicit data from or market to children under 18 years of age.` to `We do not knowingly solicit data from or market to individuals under the age of 18.`